client/src/views/users/UserDetailsModal.vue

```diff
@@ -0,0 +1,211 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="modal">
+    <ms-modal
+      :title="$t('UsersPage.UserDetailsModal.title')"
+      :close-button="{ visible: true }"
+    >
+      <div>
+        <ion-text class="text1">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.name') }}
+        </ion-text>
+        <ion-text class="text1-right">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.joined') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text2">
+          {{ user.humanHandle.label }}
+        </ion-text>
+        <ion-text class="text2-right">
+          {{ timeSince(user.createdOn, '--', 'short') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text4">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.commonWorkspaces') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-chip
+          v-if="user.isRevoked()"
+          color="danger"
+        >
+          <ion-icon
+            :icon="ellipse"
+            class="revoked"
+          />
+          <ion-label>
+            {{ $t('UsersPage.UserDetailsModal.subtitles.revoked') }}
+          </ion-label>
+        </ion-chip>
+      </div>
+      <ion-list>
+        <ion-card
+          :disabled="user.isRevoked()"
+          class="workspace-card"
+        >
+          <ion-card-content>
+            <ion-avatar class="card-content-icons">
+              <ion-icon
+                class="card-content-icons__item"
+                :icon="business"
+              />
+              <ion-icon
+                class="cloud-overlay"
+                :class="'cloud-overlay-ko'"
+                :icon="cloudOffline"
```

Comment:
What is the purpose of the second ':class' line? Here you will always have class="cloud-overlay cloud-overlay-ko" since you have no boolean condition in your line.
Here the icon never changed, aren't we supposed to switch from cloudOffline to cloudOnline (or something like this) based on some conditions?

Reply:
Indeed the icon states aren't implemented, this the version it's taken from 

```
class="cloud-overlay"
:class="workspace.availableOffline ? 'cloud-overlay-ok' : 'cloud-overlay-ko'"
:icon="workspace.availableOffline ? cloudDone : cloudOffline"
```

I'll fix it 👍